### ISSUE TITLE
Keine Menübedienung

### ISSUE THREAD

**Issue by coppa72** [modelleicher/FS19_realManualTransmission#3]
Hallo,

leider kann ich im aufgerufenem Menü keine Veränderungen vornehmen. Ich nutze Keyboardsteer und habe es beim Laden des Spielstandes deaktiviert. Brachte keinen Erfolg.

Gruß Coppa72

**Comment by modelleicher** [comment]
Hallo,
Hast du keinen Mauszeiger? Oder passiert einfach nichts wenn du auf die Schaltflächen klickst? Irgendwas in der Log? 

LG

**Comment by coppa72** [comment]
Hallo,

habe die Probleme mit dem Fendt. In der Log schreibt dieser Mod Fehler.

Hier die log.
[log.txt](https://github.com/modelleicher/FS19_realManualTransmission/files/2829702/log.txt)

Gruß Coppa72

**Comment by modelleicher** [comment]
Hallo,
Konnte in der Log keine Fehler vom RMT finden. Hast du es mal ganz ohne andere Mods probiert? 
Mit VehicleEnhancement spiele ich ja auch, das sollte gehen. Vielleicht gibt es Komplikationen mit VehicleFruitHud ?

LG

**Comment by coppa72** [comment]
Guten Morgen,

werde es am Wochenende ausprobieren. Komme vorher nicht dazu. Melde mich dann wieder.

Gruß Coppa72

**Comment by Nanuk9991** [comment]
Ich häng mich hier mal rein.
Erstmal danke für den Mod:)
VCA (KeyboardSteer) und dein Mod zusammen funktioniert nicht?

**Comment by modelleicher** [comment]
> Ich häng mich hier mal rein.
> Erstmal danke für den Mod:)
> VCA (KeyboardSteer) und dein Mod zusammen funktioniert nicht?

Habe es nicht mehr probiert seitdem KeyboardSteer VCA ist, nehme aber mal an dass die Schaltung im VCA die selben Funktionen überschreibt, somit könnte das Probleme machen. Theoretisch, wenn man die Schaltung im VCA komplett deaktivieren kann für die Fahrzeuge die RMT haben, dann könnte es gehen. Man müsste aber Shuttle dann auch komplett deaktivieren denke ich.

LG

**Comment by Nanuk9991** [comment]
Habe schon unter dein Video gepostet, leider geht es nicht, obwohl shuttle und getriebe usw. im VCA deaktiviert sind, funktioniert der Reverser bei dir dann nicht, das is sehr schade. Gründe unter deinem Video.
Ansonsten eine saubere Leistung
Top!

**Comment by CityFarmerDE** [comment]
Der LS 19 scheint da keine Doppelbelegungen zu mögen. Kann also nicht die gleiche Taste für z.B. Handbremse in RMT und VCA oder TSX nutzen. Wenn ich verschiedene Tasten belege, geht es (Habe nur TSX getestet). Es wäre schön, wenn RMT noch eine Handbremse und Shuttle Funktion für Fahrzeuge bekommt, die kein manuelles Getriebe haben, damit man nicht jeweils zwei Tasten für Shuttle und Handbremse belegen muß.
Freue mich schon auf die kommenden Versionen, so macht das spielen endlich wieder Spaß. Danke dafür

**Comment by modelleicher** [comment]
Ja, selbe Taste für unterschiedliche Belegungen beim selben Fahrzeug mag der Ls19 nicht.. Hoffe das wird gefixt im nächsten Patch, das ist echt nervig.. :)

Wegen Shuttle und Handbremse für Fahrzeuge ohne Getriebe.. Mal schauen. Der Vorschlag kam ja schon öfter. Andererseits sehe ich es aber auch irgendwie nicht als Sinnvoll an alles 5x umzusetzen, und TSX hat ja Shuttle und Handbremse schon sehr schön umgesetzt. Wenn da nicht das Tastenbelegungs-Problem wäre wäre das ja gar kein Problem..

LG

**Comment by D9005** [comment]
Hallo, 

Also bei mir hat's funktioniert wenn ich Handbremse für TSX unter Taste 1 und für RMT unter Taste 2 belegt habe. Funktioniert bei den Gamepad Belegungen genauso ;) nur bei der Maus klappt's leider nicht.

MfG D9005

**Comment by CityFarmerDE** [comment]
Danke für den Hinweis. Klappt bei mir auch. T150 Pro und Seitenkonsole. Evtl. als Hinweis in der Bechreibung ergänzen?

**Comment by modelleicher** [comment]
Das klingt ja fast nach einer Lösung für das Problem der Doppelbelegungen.. :) 

Was VCA angeht.. hab selbst mal rum probiert, selbst wenn ich alles auf Off steht geht in Fahrzeugen mit RMT nichts. wie es Nanuk beschrieben hat. Das ist natürlich schade.. mal schauen ob ich da irgendwas machen kann, aber im Moment bezweifel ich das etwas.

**Comment by Bocki1504** [comment]
Bei mir kann ich im Menü nichts anklicken Mauszeiger ist da aber es wird nichts ausgewählt:

**Comment by loeschknecht** [comment]
Deaktivier mal Courseplay. Bei mir hats geholfen.
